nunit.templates/ItemTemplates/Test/NUnit Test Fixture/TestClass.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
```

Comment:
People often have a rule that sorts System namespaces above all others. Not sure which way we should ship it.